src/main.tf

```diff
@@ -16,22 +16,12 @@ resource "azurerm_storage_account" "main" {
   account_tier                      = "Standard"
   account_kind                      = "StorageV2"
   access_tier                       = "Hot"
-  account_replication_type          = var.redundancy.replication_type
+  account_replication_type          = var.redundancy.data_protection ? var.redundancy.replication_type : "LRS"
   min_tls_version                   = "TLS1_2"
-  public_network_access_enabled     = false
+  public_network_access_enabled     = true
```

Comment:
```suggestion
  # this can be changed without forcing a recreate
  # and can potentially be changed later when
  # we try to push this into a completely private network
  public_network_access_enabled     = true
```